radish34/api/src/services/po.js

```diff
@@ -335,6 +335,9 @@ export const createPO = async (_zkpPrivateKeyOfBuyer, oldMSA, newMSA, po) => {
   } = await createPOTransaction(proofArray, publicInputHash.field(248, 1, true), publicInputs);
 
   console.log('\nCREATE PO TRANSACTION COMPLETE');
+  console.log('proofArray:', proofArray);
```

Comment:
these console.logs need to be removed

Reply:
Removed

Reply:
Well, yes probably. If they're helpful I could see leaving them in for now. However it does seem like this is sensitive information (or at least would be in a production setting). We do need to do a security review of our logs at some point (also convert them to proper logging instead of `console.log` where we can make these specific logs `DEBUG` only logs. Here's a ticket I wrote for that work effort. https://github.com/ethereum-oasis/baseline/issues/101

Reply:
Yes good point. These logs might be a good candidate to display in "debug" mode but not "production" mode